integration/src/main/java/scripts/testscripts/WorkspaceLifecycle.java

```diff
@@ -26,7 +26,6 @@
 import scripts.utils.WorkspaceApiTestScriptBase;
 
 public class WorkspaceLifecycle extends WorkspaceApiTestScriptBase {
-  private static final String USER_EMAIL = "liam.dragonmaw@test.firecloud.org";
```

Comment:
random question need to help me understand. when you remove this hard code test user email. then the test user will not be this person, will get the email from the actual tester. is that correct?

Reply:
yes, it can be different. test user is different per environment. So the test is failing in verily-src.